### DISCUSSION TITLE
Changing the route of the edge when the there is an input on left and on top

### DISCUSSION THREAD

**Discussion by SciManJD** [bcakmakoglu/vue-flow#871] [Q&A] [answered]
I created custom node, because I need two inputs. The first input is on the left side and the second on the top. When I make the connections, the line to the top input doesn't go directly straight up as I want:

![example](https://user-images.githubusercontent.com/98314127/233022965-ae89d2cf-3dbe-4a4d-885d-76437aee6661.png)

I already saw that this component has a lot of features, so I'm sure that there is a way to do this, but I couldn't find a way. Please, help!

**Comment by bcakmakoglu** [comment, marked as answer]
I revert my first statement 😄 You can do this without a custom edge.

[Here's a simple sandbox doing this.](https://codesandbox.io/p/sandbox/bold-banzai-vklxki?file=%2Fsrc%2FApp.vue)

**Reply by bcakmakoglu** [reply]
Oh, and make sure you actually define the handle position properly 

```vue
<Handle :position="Position.Right" ... />
```

Otherwise edges will fall back to the default positions.

**Reply by SciManJD** [reply]
Great... It is so, easy. Thank you for the great support!
